main.cpp

```diff
@@ -71,7 +71,7 @@ it's very easy to mess this task up.
 int main()
 {
                       //name, hit points, armor. 
-    Dwarf dwarf { "Maldin the Dwarf", 3, 1 };
+    Dwarf dwarf ( "Maldin the Dwarf", 3, 1 );
```

Comment:
This line should not need changing.
Put it back to the previous version.
if it doesn't compile, you need to figure out why it doesn't compile.
- [ ] revise